index.js

```diff
@@ -303,43 +324,86 @@ FastForEach.prototype.insertAllAfter = function(nodeOrNodeArrayToInsert, insertA
   return nodeOrNodeArrayToInsert;
 };
 
+// checks if the deleted data item should be handled with delay for a possible reuse at additions
+FastForEach.prototype.shouldDelayDeletion = function (data) {
+  return this.allowMoveNodes && (typeof data === "object" || typeof data === "function");
```

Comment:
Because as far as I know you can't add properties to primitives, and we use a custom property for fast pending deletion lookups. This check was ment to see if we can add custom property. But correct me if I'm wrong.